DiamondLightSource/SynchWeb
# LIMS-1113: Shipping service dispatch redirect

**JIRA ticket**: [LIMS-1113](https://jira.diamond.ac.uk/browse/lims-1113)

**Summary**:
Adds the option to redirect to the shipping service to book shipments when requesting dispatch of appropriate Dewars.

**Changes**:
- Adds feature flag `$use_shipping_service_redirect` which enables/disables this feature
- On requesting Dewar dispatch, create a shipment request in the shipping service and redirect to the appropriate shipment workflow to book a shipment
- Use minimised dispatch request email for shipments booked via the new pathway

**To test**:
- Check that the appropriate dispatch form is displayed for academic, industrial, international users
- Check that the appropriate shipment request is created in the shipping service when requesting dispatch of a domestic academic Dewar
- Check that the minified dispatch-lite email template is used for such dispatch requests; ensure this has an appropriate link to where a Goods Handling operative can view and print the air waybill
- Check that the dispatch button links to the shipping service once a dispatch shipment request has been successfully created.
